reframe/core/decorators.py

```diff
@@ -83,3 +93,32 @@ def _do_register(cls):
         return cls
 
     return _do_register
+
+
+def required_version(*versions):
+    """Class decorator for skipping version-uncompatible tests.
```

Comment:
Class decorator for specifying the required ReFrame version for the following test.